src/execution/gaugeExecutor.ts

```diff
@@ -315,10 +311,22 @@ export class GaugeExecutor extends Disposable {
                 return this.execute(null, new ExecutionConfig().setRepeat()
                     .setStatus(join(defaultProject.root(), "previous run"))
                     .setProject(defaultProject));
+            }),
+            commands.registerCommand(GaugeVSCodeCommands.ShowReport, () => {
+                this.openReport();
             })
         ));
     }
 
+    private openReport() {
+        let projectRoot = ProjectFactory.get(this.gaugeWorkspace.getDefaultFolder()).root();
+        let url = join(projectRoot, "/reports/html-report/index.html");
```

Comment:
`reports` directory is configurable through `gauge_reports_dir` property. If the user has configured it to something else then they won't be able to open it through vscode command palette.  The same issue will happen when `overwrite_reports` is set to `false`.

Reply:
yes, we need to handle these cases. Is it ok if separate cards are raised for the same and looked into?

Reply:
Please create a separate issue for the same.

Reply:
Done. https://github.com/getgauge/gauge-vscode/issues/362